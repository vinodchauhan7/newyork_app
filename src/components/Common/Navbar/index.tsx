import BreadCrumb from "../BreadCrumb";


type Props = {};

const Navbar = (props: Props) => {
    return (
            <header className="flex flex-col justify-center">
                <nav className="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8" aria-label="Global">
                    <h1 data-testid="heading" className='font-bold md:text-6xl text-2xl font-sans text-black-950'>The New York Times</h1>
                </nav>
                <div>
                    <BreadCrumb />
                </div>
            </header>
        );
}

export default Navbar;