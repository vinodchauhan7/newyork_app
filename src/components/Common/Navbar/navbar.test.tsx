
import Navbar from "./index"
import { render, screen} from "@testing-library/react";
import '@testing-library/jest-dom';

test("Render Navbar Component", () => {
  const {container} = render(<Navbar/>);
  const parent = screen.getByText(/The New York Times/i);
  expect(container).toMatchSnapshot();
  expect(parent).toBeInTheDocument();
})