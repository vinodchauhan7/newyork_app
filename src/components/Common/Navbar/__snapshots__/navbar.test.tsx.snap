// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Navbar Component 1`] = `
<div>
  <header
    class="flex flex-col justify-center"
  >
    <nav
      aria-label="Global"
      class="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8"
    >
      <h1
        class="font-bold md:text-6xl text-2xl font-sans text-black-950"
        data-testid="heading"
      >
        The New York Times
      </h1>
    </nav>
    <div>
      <div
        class="flex py-5 items-center"
      >
        <div
          class="flex-grow border-t-2 border-black"
        />
        <span
          class="flex-shrink mx-4 text-1xl md:text-2xl text-blue-950"
        >
          Most Popular Articles
        </span>
        <div
          class="flex-grow border-t-2 border-black"
        />
      </div>
    </div>
  </header>
</div>
`;
